tests/lambda_functions/nightly_stats_bigquery_upload/test_nightly_lambda.py

```diff
@@ -321,7 +321,10 @@ def test_lambda_handler(
         assert response == {'statusCode': 200}
 
     @staticmethod
-    def test_should_not_delete_existing_stats_from_bigquery_table_if_table_does_not_exist(mock_bigquery_client) -> None:
+    def test_should_not_delete_existing_stats_from_bigquery_table_if_table_does_not_exist(
+        mock_ssm_client,
```

Comment:
This was necessary. Would sometimes be loaded from a previous test, sometimes not.